src/components/TodoForm.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+
+const TodoForm = ({addTask}) => {
+  const [userInput, setUserInput] = useState('');
+
+  const handleChange = (e) => {
+    setUserInput(e.currentTarget.value);
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    addTask(userInput);
+    setUserInput('');
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <input
+        value={userInput}
+        type='text'
+        onChange={handleChange}
+        placeholder='Введите задачу'
+      />
+      <button>сохранить</button>
```

Comment:
add a type for the button. If you put the correct one, then you will not need e.preventDefault();